buildSrc/src/main/groovy/org/apache/beam/gradle/BeamModulePlugin.groovy

```diff
@@ -1411,8 +1413,13 @@ class BeamModulePlugin implements Plugin<Project> {
 
         project.tasks.register("jmh", JavaExec)  {
           dependsOn project.classes
-          mainClass = "org.openjdk.jmh.Main"
+          mainClass = "org.apache.beam.sdk.testutils.jmh.Main"
           classpath = project.sourceSets.main.runtimeClasspath
+
+          environment([
+            'INFLUXDB_MEASUREMENT': 'java_jmh_' + (getPath().split(':') - ['', 'sdks', 'java', 'jmh']).join('_')
```

Comment:
Why is it important to split up the JMH benchmarks under different measurements?

Reply:
Happy to discuss this... as long as benchmarks are in core (or core related modules) only I wouldn't consider this to be important. But assuming there could be benchmarks on IOs as well, it would be convenient to scope them in a more granular way. That makes it a bit easier to treat them differently.

Reply:
I was under the impression that if the data we collect for a measurement is the same then we should use a single measurement. For example we could have two different types of JMH runs (throughput and sample time) which we would want to store in a different measurement since they are collecting different data. In our case we are using the JMH throughput mode run for all our benchmarks currently so it would make sense to store it all in one measurement.

nit: note that folks may want to have benchmarks under `runners/` like `runners/java-fn-execution` which could lead to a collision in measurement names.

Reply:
Ok, that sounds perfectly reasonable to me. Thanks for the suggestion 👍 
I'll turn this into a prefix and append the label of the JMH mode, so this would basically end up being `java_jmh_thrpt`.  I was briefly considering if the measurement name should also include the `scoreUnit` (in case changed using `-tu`), but I don't think so. Regardless of the scale, semantically it's still the same type of measurement. But let me know if you disagree.

Reply:
> nit: note that folks may want to have benchmarks under runners/ like runners/java-fn-execution which could lead to a collision in measurement names.

Not really relevant anymore, but I can't see how to realistically generate duplicates with the above as the module path is unique and the exclusions are very specific &ast;wondering&ast;

Reply:
Yes very unlikely which is why I tagged it as a nit.